tests/virt_autotest/guest_upgrade_run.pm

```diff
@@ -49,19 +49,43 @@ sub analyzeResult {
             $result->{$testcase_name}->{error}  = $error             ? $error   : 'none';
         }
     }
+
+    #if there are failing guest installation
```

Comment:
Please use # followed by a single space comment style as we do.

Reply:
no problem